clearwater-cassandra/usr/share/clearwater/clearwater-cluster-manager/failed_plugins/cassandra_failed_plugin.py

```diff
@@ -58,10 +58,18 @@ def on_stable_cluster(self, cluster_view):
         pass
 
     def on_leaving_cluster(self, cluster_view):
+
+        # Cassandra commands need to be run in the signaling network
+        # namespace in split network systems.
+        def in_namespace(command):
```

Comment:
`in_sig_namespace` would be better - it can't run commands in any other namespace after all.